dotnet-test-samples/test-containers/tests/ApiTests.UnitTest/TestStartup.cs

```diff
@@ -0,0 +1,85 @@
+﻿using Amazon.DynamoDBv2;
+using DotNet.Testcontainers.Containers;
+using Testcontainers.DynamoDb;
+
+namespace ProductAPI.Tests;
+
+using Amazon.DynamoDBv2.Model;
+using Amazon.Runtime;
+
+using Microsoft.Extensions.Options;
+
+using ServerlessTestApi.Core.DataAccess;
+using ServerlessTestApi.Core.Models;
+using ServerlessTestApi.Infrastructure;
+using ServerlessTestApi.Infrastructure.DataAccess;
+
+public sealed class TestStartup : IDisposable
+{
+    private readonly IContainer container;
+
+    public IProductsDAO ProductsDao { get; }
+    
+    public Product TestProduct { get; }
+
+    public TestStartup()
+    {
+        container = new DynamoDbBuilder()
```

Comment:
I think that if we run containers as part of the test that test should not be part of the project's unit tests. It would run longer and has bigger scope and should be a part of the integration tests (emulated instead of using AWS services) sinc ethey have the same objectives.

Reply:
Yeah ok, that makes sense. So what would you call the tests I have that actually make calls against deployed resources?